src/addons/sourcemod/scripting/zr/zadmin.inc

```diff
@@ -104,7 +104,7 @@ bool ZAdminMenu(int client)
     // Add items to menu.
     SetMenuTitle(menu_zadmin, title);
     AddMenuItem(menu_zadmin, "classmultipliers", classmultipliers, configdraw);
-    AddMenuItem(menu_zadmin, "weapons", weapons, configdraw);
+    AddMenuItem(menu_zadmin, "weapons", weapons, moderatordraw);
```

Comment:
why this change

Reply:
Bcs moderatordraw  (OperationType_Generic) can already restrict weapons by command line, allow them to do it via zadmin menu
https://github.com/srcdslab/sm-plugin-zombiereloaded/blob/298a89628aab9c49c9ff0fab2dc60d7b4d49f130/src/addons/sourcemod/scripting/zr/weapons/restrict.inc#L677

Reply:
ok good point